src/sanitizer.js

```diff
@@ -26,9 +26,18 @@ export const sanitizeDocFragment = function _sanitizeDocFragment(
  * @private
  */
 export const _normalizeConfig = function _normalizeConfig(config) {
-  if (!config) {
+  if (
+    config &&
+    Object.keys(config).length === 0 &&
+    config.constructor === Object
+  ) {
     config = {};
   }
+  for (let [configurationElementList, elements] of Object.entries(config)) {
```

Comment:
This way of in-place mutating each element in each list in `config` is probably very janky 😅. Also, `Object.entries` isn't [supported by older browsers](https://caniuse.com/object-entries)

Is there a better way to do this?